geometry/eolearn/geometry/morphology.py

```diff
@@ -2,15 +2,11 @@
 Module containing tasks for morphological operations
 """
 
-import logging
-
 import skimage.morphology
 import numpy as np
 
 from eolearn.core import EOTask
 
-LOGGER = logging.getLogger(__name__)
-
```

Comment:
Is this not necessary for EOExecutor stuff?

Reply:
Maybe @AleksMat can also comment.

Reply:
@AleksMat said it's fine.